src/editor/codemirror/structure-highlighting/view.ts

```diff
@@ -64,7 +65,7 @@ export const codeStructureView = (option: "full" | "simple") =>
           depth: number,
           body: boolean
         ) => {
-          const state = view.state;
+          const hints = view.state.field(hintState).hints;
```

Comment:
This is pretty confusing - not the new hints line but that removed state line. The correct state is still in scope but defined after the nested function which is pretty confusing. I'll rejig this tomorrow.

Reply:
a3272b946d37af3110c444b47f70b9da0d0d5624